Gemfile

```diff
@@ -5,8 +5,8 @@ source "https://rubygems.org"
 ruby RUBY_VERSION
 
 gem "decidim", path: "."
-gem "decidim-consultations", path: "decidim-consultations"
-gem "decidim-initiatives", path: "decidim-initiatives"
+gem "decidim-consultations", path: "."
+gem "decidim-initiatives", path: "."
```

Comment:
@oriolgual and I had some back-and-forth with this, as it didn't work for him using `.` as a relative path. Might have to do with the ruby version manager used. @oriolgual can you confirm this works for you?

Reply:
I should work because we're doing the same for `decidim` and `decidim-dev`, no?

Reply:
I can't actually remember this 😅

But if it fails only at my computer then it's an issue with my setup, not Decidim's

Reply:
Well it might indicate a flaw with the approach. But let's do this!